libsplinter/src/network/connection_manager/mod.rs

```diff
@@ -564,7 +564,7 @@ pub mod tests {
     #[test]
     fn test_heartbeat_notifications_raw_tcp() {
         let mut transport = Box::new(RawTransport::default());
-        let mut listener = transport.listen("tcp://localhost:8080").unwrap();
+        let mut listener = transport.listen("tcp://localhost:3030").unwrap();
```

Comment:
You could use `localhost:0` and it will generate a random port.  Take a look at the tests in `libsplinter/src/transport/raw.rs`

Reply:
I'm not in love with the idea of having the port be random each run. Also this seems to cause several tests to fail and hang in OSX.